docs/conf/portland/2022/index.rst

```diff
@@ -1,5 +1,8 @@
 :template: {{year}}/index.html
 :banner: _static/conf/images/headers/{{ shortcode }}-group.png
+:og:title: Home - Write the Docs {{ name }} {{ year }}
```

Comment:
Weird this doesn't seem to be working.

```
  <meta property="og:title" content="<no title>" />
```

Wonder if we can do something like a default site title, and then overrride it IF there is a page title. Which would give us a boring WTD on every page till we fix the in page thing, but much better than no-title

Reply:
Which is probably what you're trying to do here? 
https://github.com/writethedocs/www/pull/1724/files#diff-edc5891e3455bb5cd05af781bd73215c81f277e7a78a6350e2198621d6a898c8R46